### PR TITLE
Add query params for getRoomMessages and add cursor methods

### PR DESCRIPTION
**Pull request by morrislaptop**
### Why?

----

- [ ] CHANGELOG updated if relevant?

### CODE REVIEW

File: src/Chatkit.php [hunk 1]
```diff
@@ -299,6 +301,71 @@ public function deleteRoom($options)
...
+    }
+
+    /**
+     * Sets the read cursor for a user in a room.
+     *
+     * @param array $options
+     *              [Available Options]
+     *              • user_id (string|required): Represents the ID of the user that you want to get the rooms for.
+     *              • room_id (string|required): Represents the ID of the user that you want to get the rooms for.
+     *              • position (string|required): Represents the ID of the user that you want to get the rooms for.
```
Comment (hamchapman) [inline src/Chatkit.php]: I _believe_ `position` needs to be an integer, but maybe the server is being "forgiving" and allowing strings as well? Or maybe this is just needs a documentation tweak 😊

**Review by hamchapman — APPROVED** [review]
This is awesome - thanks! 

Just one comment about `position` for a cursor being an integer and not a string 👍

**Comment by morrislaptop** [comment]
@hamchapman updated docs, thanks!

**Comment by morrislaptop** [comment]
@hamchapman merge?

**Comment by morrislaptop** [comment]
Release? :)

**Comment by hamchapman** [comment]
0.5.5 released. I changed the name of `setCursor` to `setReadCursor` to be consistent with `getUserReadCursors` (as opposed to changing that to be `getUserCursors`, as we might support different cursor types in future)

Thanks again! 👍

**Pull request merged**